wordpress-mobile/WordPress-iOS
# Issue/8054 fix download of images org basic auth

Fixes #8054 

To test:
 - Launch the app and add a self-hosted site with basic auth enabled
 - Go to Media Library, check that images download correctly
 - Tap an image and see if it opens correctly
 - Edit a post add an image to it, save it and reopen again and see if image loads correctly
 - Add a feature image image to the same post see if it saves and loads correctly on post settings
 - Check if in the post list the images are loaded correctly.
